test/Benchmarks/src/Utils.enso

```diff
@@ -0,0 +1,26 @@
+private
+
+from Standard.Base import all
+
+## Find enso binary inside `built-distribution` directory.
+find_enso_bin =
+    find_prefix dir prefix =
+        vec = dir.list name_filter=prefix+"*"
+        if vec.length == 1 then vec.at 0 else
+            msg = "Cannot find " + prefix + "* in " + dir.to_text + '\n'
+            err = dir.list.fold msg t-> f->
+                t + f.to_text + '\n'
+            Panic.throw err
+
+    project_root = File.new enso_project.root.to_text
+    repository_root = project_root . parent . parent
+    built_distribution = find_prefix repository_root "built-distribution"
+    enso_engine = find_prefix built_distribution "enso-engine-"
+    enso = find_prefix enso_engine "enso-"
+    bin = find_prefix enso "bin"
+
+    exe = File.new bin / if Platform.os == Platform.OS.Windows then "enso.bat" else "enso"
```

Comment:
Sometimes it is `enso.bat`, sometimes it is `enso.exe`.

Reply:
This is just copied from what was already there. Which explains why it has worked before (on the CI) - benchmarks are only executed on Linux.